setup.py

```diff
@@ -43,7 +43,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket >= 1.13.2",
+        "pytket >= 1.13.2, <2.0",
```

Comment:
Hope this is fine with the changes? Wanted to make sure that we are not going to use 2.x in the future

Reply:
Looks fine to me!